content/docs/apps/managing-teammates.md

```diff
@@ -23,18 +23,30 @@ Org Managers can assign roles using the [dashboard](https://dashboard.fr.cloud.g
 
 {{% dashboard %}}
 
+### Organization users
+
 If you're an Org Manager, go to your [dashboard](https://dashboard.fr.cloud.gov/) and click on the name of your org.
 
 ![User roles](/img/user-role-management.png)
 
 On the org page, scroll down to "Organization users". This lists the org users and their org roles. Change the roles by clicking the checkboxes beside their names. (If you're the only person in the org, you'll just see "You are the only user in this organization".)
 
-To give a teammate access to a space, go to your org page, click a space name, and scroll down to "Space users". You can add a person, and then you can give them roles with the checkboxes.
+### Space users
+
+If you're an Space Manager, go to your [dashboard](https://dashboard.fr.cloud.gov/) and click on the name of your org, then click on the name of your space.
+
+![Space roles](/img/space-user-role-management.png)
+
+On the space page, scroll down to "Space users". This lists the space users and their space roles. Change the roles by clicking the checkboxes beside their names. (If you're the only person in the space, you'll just see "You are the only user in this space".)
+
+To give a organization user access to a space, go to your space page, click a space name, and scroll down to "Space users". You can add a person, and then you can give them roles with the checkboxes.
```

Comment:
`To give a organization user` -> `To give an organization user` or `To give an existing organization user`